randovania/games/dread/json_data/Hanubia.txt

```diff
@@ -1012,7 +1016,7 @@ Extra - asset_id: collision_camera_015
   > Dock to EMMI Zone Exit East (dooremmy_003)
       Before Escape Sequence
   > Tile Group (POWERBOMB) 2
-      Trivial
+      Simple IBJ or Use Spin Boost
```

Comment:
On this one, I probably could, but since it's not relevant to escape sequence I'd rather do it while looking over Hanubia as a whole, will keep it noted for then though.